manager/integration/tests/test_csi.py

```diff
@@ -51,7 +51,10 @@ def create_pv_storage(api, cli, pv, claim, backing_image, from_backup):
         numberOfReplicas=int(pv['spec']['csi']['volumeAttributes']
                              ['numberOfReplicas']),
         backingImage=backing_image, fromBackup=from_backup)
-    common.wait_for_volume_restoration_completed(cli, pv['metadata']['name'])
+    if from_backup:
```

Comment:
I am curious why this worked before. It should fail as well, right?

Reply:
This only fails from 2 days ago. After the function explicitly waits for restoration started in this commit: https://github.com/longhorn/longhorn-tests/pull/1344/files#diff-a2154691ec50c6925b7f385cf30c42f33b2a3c935192c894bfe4b32fe3ee49dcR4309